ui/src/tasks/components/TaskRunsPage.tsx

```diff
@@ -43,7 +44,9 @@ class TaskRunsPage extends PureComponent<Props> {
             <Page.Header.Left>
               <Page.Title title="Runs" />
             </Page.Header.Left>
-            <Page.Header.Right />
+            <Page.Header.Right>
+              <Button onClick={this.handleRunTask} text="Run Task" />
```

Comment:
Would be rad to add a play icon to this button
```suggestion
              <Button onClick={this.handleRunTask} text="Run Task" icon={IconFont.Play} />
```

Reply:
Got it! Thanks :)